File: app/src/main/java/com/github/chagall/lighthouse/LinkModel.java
package com.github.chagall.lighthouse;

public class LinkModel {
    public String sender;
    public String link;
    public String originalMessage;

    public LinkModel(String sender, String link, String originalMessage) {
        this.sender = sender;
        this.link = link;
        this.originalMessage = originalMessage;
    }
}

File: app/src/main/java/com/standford/ligthhouse/model/LinkModel.kt
package com.standford.ligthhouse.model

import java.util.*

data class LinkModel(
    var sender: String,
    var link: String,
    var originalMessage: String,
    var identifier: String,
    var topline: String,
    var rank: String,
    var score: String,
    var writeup: Any?,
    val messagesContainingDomain: ArrayList<MessageModel> = ArrayList()
)